go.mod

```diff
@@ -43,12 +43,14 @@ require (
 	go.uber.org/multierr v1.4.0 // indirect
 	golang.org/x/crypto v0.0.0-20210506145944-38f3c27a63bf // indirect
 	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	google.golang.org/api v0.28.0
+	golang.org/x/oauth2 v0.0.0-20210622215436-a8dc77f794b6
+	google.golang.org/api v0.30.0
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.21.0
 	k8s.io/apimachinery v0.21.0
 	k8s.io/client-go v0.21.0
 	k8s.io/klog/v2 v2.8.0
 	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10
 )
+
+replace google.golang.org/grpc => google.golang.org/grpc v1.29.1
```

Comment:
Can't you use the latest version?

Reply:
`github.com/openshift/library-go` has hard dependency on this version